Try to enforce openblas feature.

jeromekelleher opened the pull request:
<!--
Thank you for pull request.
Below are a few things we ask you kindly to self-check before getting a review. Remove checks that are not relevant.
-->
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [ ] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

More efforts to force openblas rather than mkl.

<!--
Please note any issues this fixes using [closing keywords]( https://help.github.com/articles/closing-issues-using-keywords/ ):
-->

<!--
Please add any other relevant info below:
-->

jeromekelleher committed:
Try to enforce openblas feature.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe```) and found it was in an excellent condition.

jeromekelleher committed:
Disable blas requirement on Windows.

We get an exception:

2019-02-01T16:59:57.4180116Z conda.exceptions.UnsatisfiableError: The
following specifications were found to be in conflict:
2019-02-01T16:59:57.4180321Z   - blas==1.1=openblas
2019-02-01T16:59:57.4180489Z   - numpy=1.11 -> blas==1.0=mkl

Looks like the numpy version here depends on mkl.

jeromekelleher commented:
Not clear here whether the package should explicitly declare the openblas feature here or whether requiring the package is enough. The conda documentation of this has disappeared, so it's hard to know.
